docs/docs/configuration/detectors.md

```diff
@@ -77,3 +77,33 @@ detectors:
 ```
 
 When using CPU detectors, you can add a CPU detector per camera. Adding more detectors than the number of cameras should not improve performance.
+
+## OpenVino
+
+The OpenVino detector allows Frigate to run an OpenVino IR model on Intel CPU, GPU and VPU hardware.
+
+### OpenVino Devices
+
+The OpenVino detector supports the Intel-supplied device plugins and can specify one or more devices in the configuration.  See OpenVino's device naming conventions in the [Device Documentation](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Working_with_devices.html) for more detail. Other supported devices could be `AUTO`, `CPU`, `GPU`, etc.
```

Comment:
I assume you may be waiting for results from your test, but just confirming you intend to add a section for the `VPU` NCS2 if it works out?

Reply:
Yes, I didn't want to add documentation I had no way of verifying was correct.